river/metrics/cluster/calinskiharabasz.py

```diff
@@ -0,0 +1,125 @@
+import math
+
+from river import stats, utils
+
+from . import base
+
+__all__ = ["CalinskiHarabasz"]
+
+
+class CalinskiHarabasz(base.InternalClusMetric):
+    """Calinski-Harabasz index (CH).
+
+    The Calinski-Harabasz index (CH) index measures the two criteria simultaneously
+    with the help of average between and within cluster sum of squares. The numerator
```

Comment:
I am not sure I follow:

"(...) measures the two criteria" -> "(...) measures two criteria"?

Are these criteria the numerator and denominator described after this sentence?

Reply:
Yes indeed, two criteria taken into account by the CH index are the numerator and denominator described in the following sentences. To make it clearer, I can change the wording as you suggested, while separating the two criteria into bullets.